app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
+  def show
+    @user = User.find(params[:id])
+  end
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render 'new'
+    end
+  end
```

Comment:
こんな感じでメソッド感に改行入れてもらったほうが違いがわかるからありがたいかな！

```suggestion

  def show
    @user = User.find(params[:id])
  end
  
  def create
    @user = User.new(user_params)
    if @user.save
      flash[:success] = "Welcome to the Sample App!"
      redirect_to @user
    else
      render 'new'
    end
  end
```

Reply:
承知しました！
以後、気をつけます